narwhals/_arrow/series.py

```diff
@@ -1303,6 +1303,14 @@ def ordinal_day(self: Self) -> ArrowSeries:
             pc.day_of_year(self._compliant_series._native_series)
         )
 
+    def weekday(self: Self) -> ArrowSeries:
+        import pyarrow as pa
+        import pyarrow.compute as pc
+
+        return self._compliant_series._from_native_series(
+            pc.add(pc.day_of_week(self._compliant_series._native_series), pa.scalar(1))
+        )
```

Comment:
You can directly use the `count_from_zero` flag:

```suggestion
        return self._compliant_series._from_native_series(
            pc.day_of_week(self._compliant_series._native_series, count_from_zero=False)
        )
```